test/esp/integration/json_api_format_integration_test.rb

```diff
@@ -11,12 +11,15 @@ class JsonAPIFormatTest < ESP::Integration::TestCase
 
             assert_not_nil alert.attributes['external_account']
             assert_equal alert.external_account_id, alert.external_account.id
-            assert_not_nil alert.external_account.attributes['organization']
-            assert_equal alert.external_account.organization_id, alert.external_account.organization.id
+
+            assert_nil alert.attributes['team']
             assert_not_nil alert.external_account.attributes['team']
             assert_equal alert.external_account.team_id, alert.external_account.team.id
+
+            assert_nil alert.attributes['organization']
             assert_not_nil alert.external_account.team.attributes['organization']
```

Comment:
This test doesn't really confirm it comes back in the original request does it? It could just be making another request.